pulsar/README.md

```diff
@@ -0,0 +1,47 @@
+# Apache Pulsar Initialization Action
+
+Here we've got an initialization action that automates the installation of a single-cluster pulsar instance on bare metal. More detail can be found in the Pulsar documentation here: https://pulsar.apache.org/docs/en/deploy-bare-metal/
```

Comment:
Can we replace the first sentence with: This initialization action installs [Apache Pulsar](https://pulsar.apache.org/) on Dataproc clusters.

Reply:
ack. done!